libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -749,7 +818,13 @@ void AC_Avoid::adjust_velocity_polygon(float kP, float accel_cmss, Vector2f &des
 
     // for stopping
     const float speed = safe_vel.length();
-    const Vector2f stopping_point_plus_margin = position_xy + safe_vel*((2.0f + margin_cm + get_stopping_distance(kP, accel_cmss, speed))/speed);
+    Vector2f stopping_point_plus_margin; 
+    if (!desired_vel_cms.is_zero()) {
+        stopping_point_plus_margin = position_xy + safe_vel*((2.0f + margin_cm + get_stopping_distance(kP, accel_cmss, speed))/speed);
```

Comment:
what append line 866 if stopping_point_plus_margin is zero ? (default value)

Reply:
Should probably also check that speed is not zero, I actually had add that check in a bunch of places no we cant just return instantly for zero desired velocity